lib/modules/zfs.go

```diff
@@ -0,0 +1,60 @@
+package modules
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/davidscholberg/go-i3barjson"
+	"os/exec"
+	"strings"
+)
+
+// Zfs represents the configuration data for the ZFS block
+type Zfs struct {
+	BlockConfigBase `yaml:",inline"`
+	PoolName        string `yaml:"zpool_name"`
+	ZpoolBin        string `yaml:"zpool_bin"`
```

Comment:
I'm not sure how I feel about having the command name in a config variable. I'm guessing the reason you added this is so that you can have the user specify the full path to the binary in case it's not in the user's path, but I would say that a better solution to that is to require the user to update their path so that zpool can be called without needing a full path.